.pre-commit-config.yaml

```
@@ -0,0 +1,40 @@
+repos:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)